### DISCUSSION TITLE
Locale currency support using browser's native function

### DISCUSSION THREAD

**Discussion by rohit-innodeed** [grafana/grafana#54195] [Dashboards, Panels & Visualizations]
As far as I know, grafana does not yet have translation files/localization support as others apps provide for showing time, currency and other measurements e.g., Youtube. If I have English(India) as primary language in my browser, then Youtube shows number of likes and subscribers in crores or lacs (i.e., in Indian Currency Format) as shown in images below:

![image](https://user-images.githubusercontent.com/101693459/186577199-bb856b9f-e38b-45cc-8ffe-12c32f8b24a0.png)
![image](https://user-images.githubusercontent.com/101693459/186577230-6afd4718-be19-444b-9f12-55f7dd8aed21.png)

Grafana's currency support is more basic and straightforward. It will be good if it can show the currencies in locale currency format. Because we don't use millions and billions for large values here. We have lacs and crores. Please refer this [link ](https://en.wikipedia.org/wiki/Crore)for more information about Indian Currency format. 

![image](https://user-images.githubusercontent.com/101693459/186577884-5fb2b4ab-d69e-447c-ba48-732dd823ca31.png)
![image](https://user-images.githubusercontent.com/101693459/186577949-b16ec782-6ec8-4122-91fe-e91b6c88e06f.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
